### DISCUSSION TITLE
Combine ipv4 and ipv6 address and assign it to one device in top clients list

### DISCUSSION THREAD

**Discussion by jimmysofat6864** [AdguardTeam/AdGuardHome#2815] [Q&A]
When I open the web portal and go to top clients, it shows duplicates of my devices where one of them is the ipv4 address for my phone and the other is the ipv6 address of the same device. It would be nice if they were combined together and showed up as only one device.

**Comment by ainar-g** [comment]
Hello.  You can create a persistent named client for your device with two identifiers on the Settings / Client settings page.  Would that fix the issue for you?

**Comment by jimmysofat6864** [comment]
Probably not because my iPhone makes a new mac address for privacy reasons
every time in IOS 14. Also because it shows up as "WHOIS" lookups instead
of the actual ipv6 address of the device. The ipv4 address for each device
is an "ARP" but the source for ipv6 address devices are "WHOIS"

**Reply by ameshkov** [reply]
As I recall, iPhone generates new Mac address for new networks, it does not do it for every connection.

**Reply by twcau** [reply]
You can switch this feature off for known networks on your iOS device:

`Settings > Wi-Fi > (select network if connected or available￼) > Private Address = Off.`

This is a per-network settings, and on a home network you maintain or control where AdGuard is running - there's no need to have Private Address turned On.
